docs/kpod.1.md

```diff
@@ -82,6 +82,9 @@ Removes one or more locally stored images
 ### save
 Save an image to docker-archive or oci
 
+### rename
+Rename a container
+
```

Comment:
It looks like we're trying to maintain the sorting order here, so this belongs between `push` and `rmi`.